src/it/test_coverage_minima/validate.groovy

```diff
@@ -0,0 +1,44 @@
+def checkModule(logText, module, coverageLog) {
+    assert new File(basedir, module + "/target/scoverage.xml").exists()
+    assert new File(basedir, module + "/target/site/scoverage/index.html").exists()
+    def entry = logText.find {
+        it.startsWith("scoverage-maven-plugin:") &&
```

Comment:
i've noticed that with maven 3.9.6 ( i have that one locally) this check fails, as seems like maven 3.9 changed format from `[INFO] --- scoverage-maven-plugin:2.0.1-SNAPSHOT:check (default-cli) @ module01 ---`
to `[INFO] --- scoverage:2.0.1-SNAPSHOT:check (default-cli) @ module01 ---`
so changing to `it.startsWith("scoverage") &&` seems fine for both 3.9.x and 3.8.x

Reply:
will do